src/components/InfoPanel/InfoPanel.test.js

```diff
@@ -31,6 +31,22 @@ describe("Info Panel", () => {
     expect(wrapper.find("Topology").length).toBe(1);
   });
 
+  it("renders the expanded topology on click", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/models/group-test"]}>
+          <TestRoute path="/models/*">
+            <InfoPanel />
+          </TestRoute>
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("Modal").length).toBe(0);
```

Comment:
This test is brittle to more models in the view. I would suggest adding `data-test` id to the model and use that to ensure that is the one you are targetting.

Reply:
done